yb-voyager/src/tgtdb/conn_pool.go

```diff
@@ -121,6 +126,29 @@ func (pool *ConnectionPool) getNextUriIndex() int {
 }
 
 func (pool *ConnectionPool) setSessionVars(conn *pgx.Conn) error {
+	sessionVarsPath := filepath.Join("/", "etc", "ybSessionVariables.sql")
+	if utils.FileOrFolderExists(sessionVarsPath) {
+		varsFile, err := os.Open(sessionVarsPath)
+		if err != nil {
+			return err
+		}
+		defer varsFile.Close()
+		fileScanner := bufio.NewScanner(varsFile)
+
+		var curLine string
+		setVarRegex := regexp.MustCompile(`(?i)^SET `)
+		for fileScanner.Scan() {
+			curLine = strings.TrimSpace(fileScanner.Text())
+			if !setVarRegex.MatchString(curLine) {
+				utils.ErrExit("Only SET statements allowed in ybSessionVariables.sql. Found: %s.", curLine)
+			}
+			_, err := conn.Exec(context.Background(), curLine)
+			if err != nil && !strings.Contains(err.Error(), "unrecognized configuration parameter") {
+				return err
+			}
+		}
+		return nil
+	}
 	for k, v := range pool.params.SessionVars {
```

Comment:
Once you add the two statements to the ybSessionVariables.sql file, we no longer need this loop.

Reply:
I am keeping this loop as a failsafe in the corner case that the file does not exist for some reason